src/cf-vars.less

```diff
@@ -72,33 +72,33 @@
 // Color variables
```

Comment:
Maybe add a quick note up here explaining what the various `$color-` comments below are referring to?